Tests/UnitTests/Paywalls/Components/Properties/MaskShapePropertyTests.swift

```diff
@@ -38,19 +38,6 @@ class MaskShapePropertyTests: TestCase {
         )
     }
 
-    func testPill() throws {
-        let json = """
-        {
-            "type": "pill",
-        }
-        """
-
-        _ = try JSONDecoder.default.decode(
-            PaywallComponent.MaskShape.self,
-            from: json.data(using: .utf8)!
-        )
-    }
```

Comment:
is this removal intentional?

Reply:
Yeah yeah! `pill` type is no longer supported and the test were behind compiler flag so it didn't get caught by CI for a while 🫠